spec/support/database.rb

```diff
@@ -30,8 +30,7 @@ def setup
     end
     log = Logger.new(STDERR)
     # log = Logger.new('db.log')
-    # log.sev_threshold = Logger::DEBUG
-    log.level = Logger::Severity::UNKNOWN
+    log.level = ENV["SQL"].to_s.match?(/tr/i) ? Logger::Severity::DEBUG : Logger::Severity::UNKNOWN
```

Comment:
I'm just curious about the "tr" check?  Are you expecting "true"?  If so match might be too loose, and perhaps .start_with would be cleaner. But why not just check presence? (I like to pass "1" sometimes)

Reply:
Thanks for this comment.
I deleted this code.

Tested with no supports file, and with a logger in supports. All works as expected for me.